src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/TypeForwarders.cs

```diff
@@ -1353,6 +1353,29 @@ public class Inner
             CheckForwarderEmit(source1, source2, "NS.Forwarded", "NS.Forwarded+Inner");
         }
 
+        [ClrOnlyFact]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/79894")]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/78963")]
+        public void Extensions_01()
+        {
+            var source1 = @"
+public static class Extensions
+{
+    extension(int)
+    {
+    }
+}";
+
+            var source2 = @"
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(Extensions))]
+";
+
+            // https://github.com/dotnet/roslyn/issues/78963 - The grouping and marker types should be among the forwarded types since they are public.
+            //                                                 They also should be among exported types for library built from source1. Type symbols
+            //                                                 representing extensions from the language perspective should not be in either set.
+            CheckForwarderEmit(source1, source2, "Extensions");
```

Comment:
So the extension members are not forwarded in this temporary PR? That sounds like that wouldn't actually "unblock" anyone since they would get a similar effect just by removing the TypeForwardedToAttribute